packages/webpack/src/plugins/plugin.ts

```diff
@@ -175,12 +176,14 @@ async function createWebpackTargets(
 }
 
 function normalizeOutputPath(
-  outputPath: string | undefined
+  outputPath: string | undefined,
+  projectRoot: string
 ): string | undefined {
-  if (!outputPath) return undefined;
-  if (isAbsolute(outputPath)) {
-    return `{workspaceRoot}/${relative(workspaceRoot, outputPath)}`;
-  } else {
-    return outputPath;
-  }
+  const outputPathRelativeToWorkspaceRoot =
+    outputPath && isAbsolute(outputPath)
+      ? relative(workspaceRoot, outputPath)
+      : outputPath;
+  return projectRoot === '.'
+    ? `{workspaceRoot}/${outputPathRelativeToWorkspaceRoot}`
+    : `{projectRoot}/${outputPathRelativeToWorkspaceRoot}`;
```

Comment:
is this correct?
we have it the other way round in `vite` and `nuxt`, and and if we're using `relative` maybe it should be `join()`?

Reply:
Hmm, I was getting some error saying `{projectRoot}` is invalid. Let me check again.